src/main/scala/uk/gov/nationalarchives/XMLCreator.scala

```diff
@@ -7,6 +7,7 @@ import scala.xml._
 
 class XMLCreator {
   private val prettyPrinter = new PrettyPrinter(200, 2)
+  private val opexSchema = "http://www.openpreservationexchange.org/opex/v1.2"
```

Comment:
👍 for this approach. Being pedantic, could we call this `opexNamespace` instead? It would be fantastic if the OPEX schema was hosted at this URL though.

Reply:
Ahh, I thought it was. Cool.